brainglobe_registration/preprocessing/mask_generator.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from bg_atlasapi import BrainGlobeAtlas
```

Comment:
```suggestion
from brainglobe_atlasapi import BrainGlobeAtlas
```

`bg_atlasapi` is the old name for this package. It is now `brainglobe_atlasapi`.